templates/taxi/manufacturer_list.html

```diff
@@ -1,15 +1,21 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>
+      Manufacturer List <br>
     </h1>
+    <h3>
+      <a href="{% url 'taxi:manufacturer-create' %}">Create 🚕</a>
```

Comment:
The same

Reply:
![1](https://user-images.githubusercontent.com/111378604/200277783-56b89861-4584-496f-97ac-301d1d057c72.png)
![2](https://user-images.githubusercontent.com/111378604/200277784-1d331143-0f39-4093-8af5-49ccf0fe499f.png)
![3](https://user-images.githubusercontent.com/111378604/200277786-fc3a4b1c-17f4-4b7d-b918-47e1c730bad2.png)